src/utils/round.js

```diff
@@ -1,10 +1,25 @@
+import { isNumber } from './unit-conversion'
+
 /*
- * Round a number to the specified precision (negative precision is tens places,
- * 0 is to an integer, positive precision is to decimal places).
+ * Round a number, an array of numbers, or an Object map of numbers to the specified
+ * precision (negative precision is tens places, 0 is to an integer, positive precision
+ * is to decimal places).
  */
 export function round (number, precision = 0) {
-  const factor = Math.pow(10, precision)
-  const temp = number * factor
-  const roundedTemp = Math.round(temp)
-  return roundedTemp / factor
+  let rounded
+
+  if (Array.isArray(number)) {
+    rounded = number.map(n => round(n, precision))
+  } else if (number === Object(number)) {
```

Comment:
Nice solution. But it is a little bit hard to understand this condition, if you don't know what `Object` function do. So please use `typeof`, it would be more clear.

Reply:
`null === Object(null)` is false
`typeof null === 'object'` is true
but at the same time, `function a() {}; a === Object(a)` is also true

I'll tweak the checks